restaurants/filters.py

```diff
@@ -0,0 +1,58 @@
+import django_filters
+from .models import Restaurant
+
+class RestaurantFilter(django_filters.FilterSet):
+    title = django_filters.CharFilter(lookup_expr='icontains')
+    cost_order = django_filters.ChoiceFilter(
+        choices=[
+            ('', 'Sort by cost'),
+            ('low_to_high', 'Cost: Low to High'),
+            ('high_to_low', 'Cost: High to Low'),
+        ],
+        method='filter_by_cost',
+        empty_label=None
+    )
```

Comment:
If you use ordering filter you won't need a filter method.

```suggestion
    cost_order = django_filters.OrderingFilter(
        choices=(
            ('cost_for_two', 'Cost: Low to High'),
            ('-cost_for_two', 'Cost: High to Low'),
        )
    )
```